docs/training.md

```diff
@@ -37,6 +37,30 @@ $ cat weights
 hello train
 ```
 
+You can also use classes if you want to run many model trainings and save on setup time, this works the same way as the [Predictor](./python.md) class with the only difference being the `train` method.
```

Comment:
I think this could this could two sentences. So, "....on setup time. This works the same way..."

Reply:
Resolved in [46b12fd](https://github.com/replicate/cog/pull/2359/commits/46b12fd30ffed36261d3e084ddbbdf005297885c)